luaui/Widgets/unit_cloak_firestate.lua

```diff
@@ -58,6 +58,19 @@ end
 
 local decloakFireState = {} --stores the desired fire state when decloaked of each unitID
 
+local function GiveNotifyingOrderToUnit(uID, cmdID, cmdParams, cmdOpts)
```

Comment:
```suggestion
local function giveNotifyingOrderToUnit(uID, cmdID, cmdParams, cmdOpts)
```
Start locals with lowercase